src/index.html

```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend >Personal information</legend>
+        <label class="personal_information_label">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="personal_information_label">
+          Name:
+          <input
+          type="text"
+          name="name"
+          id="name"
+          autocomplete="off"
+          minlength="3"
+          maxlength="10"
+          >
+        </label>
+        <label class="personal_information_label">
+          How old are You?
+          <input
+          type="number"
+          name="years-old"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+        <label class="personal_information_label">
+          Full date of birth:
+          <input
+          type="date"
+          name="dateOfBirth"
+          min="1923-01-01"
+          max="2022-01-01"
+          >
+        </label>
+        <label class="personal_information_label1">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="terms"
+          min="1923-01-01"
+          max="2022-01-01"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="registration">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label class="registration1">
+          Password:
+          <input
+          type="password"
+          name="password"
+          id="password"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="facts">
+          Do you love cats?
+          <input
+          type="radio"
+          name="cat"
+          value="Yes"
+          required
+          >
+          Yes
+          <input
+          type="radio"
+          name="cat"
+          value="No"
+          required
+          >
```

Comment:
let's cover all inputs in the separate labels to avoid this behavior

Uploading Screen Recording 2023-04-19 at 10.06.29.mov…